src/config/static/markets.ts

```diff
@@ -0,0 +1,673 @@
+/* 
+  This files is used to pre-build data during the build process.
+  Avoid adding client-side code here, as it can break the build process.
+
+  However, this files can be a dependency for the client code.
+*/
+
+import { parse } from "date-fns";
+
+import { ARBITRUM, AVALANCHE, AVALANCHE_FUJI } from "./chains";
+
+const p = (date: string) => parse(date, "dd MMM yyyy", new Date());
+
+export const DEFAULT_LISTING = p("01 Jan 1970");
+
+type MarketUiConfig = {
+  listingDate: Date;
+  tokens: {
```

Comment:
I disagree with `.tokens` nesting because:

* we already have plain structures with tokens, e.x. `MarketInfo` and I didn't notice any struggle with tokens aligned with other data

* we any have index**Token**Address in each field name, so it should rather be `indexAddress`/`index` which condtradicts how we do it in other places

Reply:
Let's explicitly have `enabled: boolean` field, and use it in `MARKET_INDEX` map

Reply:
Migrated to the flat structure and added enabled key